### DISCUSSION TITLE
[FIDI] bookingDate & valueDate

### DISCUSSION THREAD

**Discussion by MacPaille** [firefly-iii/firefly-iii#5693] [Firefly III]
Hey James,

By looking at the transactions I get from Nordigen with my bank I see I have 2 dates fields: bookingDate & valueDate
I think bookingDate is used to populate the date of the transactions but I don't think valueDate is used. Shouldn't this be the Payment date field in the transactions?

**Comment by JC5** [comment]
Yes, that could work. Keep in mind this is the kind of change that will break duplicate detection 😉

**Reply by MacPaille** [reply]
Haha indeed 😄  But I'm used to it now 🤣

**Reply by JC5** [reply]
Done, in the develop branch!
